modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,194 @@
+Support Role ARN:           arn:aws:iam::123456789012:role/ManagedOpenShift-Support-Role
+Instance IAM Roles:
+ - Master:                  arn:aws:iam::123456789012:role/ManagedOpenShift-ControlPlane-Role
+ - Worker:                  arn:aws:iam::123456789012:role/ManagedOpenShift-Worker-Role
+Operator IAM Roles:
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-machine-api-aws-cloud-credentials
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-image-registry-installer-cloud-creden
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-ingress-operator-cloud-credentials
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent
+State:                      pending (Waiting for OIDC configuration)
+Private:                    No
+Created:                    Jun 10 2021 15:47:56 UTC
+Details Page:               https://cloud.redhat.com/openshift/details/s/<subscription_id>
+OIDC Endpoint URL:          https://rh-oidc.s3.us-east-1.amazonaws.com/<cluster_id>
+----
++
+[NOTE]
+====
+If installation fails or the `State` field does not change to `ready` after 40 minutes, check the installation troubleshooting documentation for more details.
+====
+. Enter the following command to follow the OpenShift installer logs to track the progress of your cluster creation:
++
+[source,terminal]
+----
+$ rosa logs install --cluster=<cluster_name> --watch
+----
++
+. Complete setup of the OIDC access-based IAM roles.
+You can complete these steps once the `State` field for the cluster creation transitions to `pending`.
+.. Create the OIDC provider.
++
+[source,terminal]
+----
+$ thumbprint=$(openssl s_client \
+  -servername rh-oidc.s3.us-east-1.amazonaws.com/${cluster_id} \
+  -showcerts \
+  -connect rh-oidc.s3.us-east-1.amazonaws.com:443 </dev/null 2>&1|
+  openssl x509 \
+  -fingerprint \
+  -noout |
+  tail -n1 |
+  sed 's/SHA1 Fingerprint=//' |
+  sed 's/://g'
+)
+
+$ aws iam create-open-id-connect-provider \
+  --url https://rh-oidc.s3.us-east-1.amazonaws.com/${cluster_id} \
+  --client-id-list openshift sts.amazonaws.com \
+  --thumbprint-list ${thumbprint}
+----
++
+[NOTE]
+====
+If the certificate changes, you will need to update the thumbprint.
+====
+
+.. Generate permissions for OIDC-access-based roles
+ Extract credential requests from the desired {product-title} version:
++
+[source,terminal] 
+----
+$ mkdir -p credrequests
+$ oc adm release extract quay.io/openshift-release-dev/ocp-release:${version:0:3}.0-x86_64 \
+    --credentials-requests \
+    --cloud=aws \
+    --to credrequests
+$ cat credrequests/0000*.yaml > credrequests/${version:0:3}.yaml
+$ rm -f credrequests/0000*.yaml
+----
+.. Create the IAM roles.
++
+[source,terminal]
+----
+$ mkdir -p iam_assets
+$ cd iam_assets
+
+$ ccoctl aws create-iam-roles \
```

Comment:
The above command had changed dir to another one. And no credrequests under it. So the command should be changed to to use the parent dir.
`--credentials-requests-dir ../credrequests/`

Reply:
This is still not fixed. And seems no usage with directory `iam_assets` at all. Why not just remove the step of the folder creation and `cd iam_assets` action?